_pages/en_US/sd-card-setup.md

```diff
@@ -12,7 +12,29 @@ Make sure to backup your SD card contents BEFORE following this. Your SD card wi
    <h2>Windows</h2>
 </noscript>
 
-## Section I - Formatting your SD card
+## Section I - Formatting your SD card with SD Formatter
+
+This section formats the SD card to the specifications by the SD Card Association. This can fix many issues that may occur with running homebrew applications.
+{: .notice--info}
+
+Any 64GB or larger SD cards will be formatted to `exFAT` in this process. You _must_ follow Section II to re-format to `FAT32`.
+{: .notice--danger}
+
+1. Download the latest version of [SD Formatter](https://www.sdcard.org/downloads/formatter/sd-memory-card-formatter-for-windows-download/)
+  - Accept the End User License Agreement to start the download
+1. Run `SD Card Formatter` (the `.exe` file) in the downloaded `.zip` file with Adminstrator privileges, then install the program
+  - The version number may be different, but it will install the same program
```

Comment:
Has a version number been mentioned? This line seems kinda random as I don't see a version being mentioned anywhere

Reply:
The .exe file has a version number. I actually tried to rewrite the line trying to get around it, and maybe I succeeded?

Reply:
I would maybe change it to `SD Card Formatter Setup` but otherwise yeah I think that works fine.